### PR TITLE
add FAQ to readme for noobies :)

### PR DESCRIPTION
**Pull request by xiudeveloper**

### CODE REVIEW

**Comment by xiudeveloper** [comment]
plz merge i took my sweet little time :)

File: README.md [hunk 1]
```diff
@@ -6,3 +6,14 @@ Strong hack
 Note:
 
 MacOS is unsupported and will NOT be fixed since it is an Apple issue.
+
+---
+
+# FAQ
+Before opening an issue, please consider the following:  
+- Is OptiFine installed? (If yes, disable it. If no, install it)
```
Comment (Luna5ama) [inline README.md:14]: I don't think Optifine causes issues with TrollHack?

Comment (Luna5ama) [reply README.md:14]: Any logs or infos on it?

Comment (xiudeveloper) [reply README.md:14]: nah didnt really think too much of it, it works now

Comment (Luna5ama) [reply README.md:14]: Then you should remove that line

**Pull request merged**